src/firefox/manifest.json

```diff
@@ -36,7 +36,8 @@
     "nativeMessaging",
     "storage",
     "tabs",
-    "webNavigation"
+    "webNavigation",
+    "dns"
```

Comment:
We need to make sure that the user is not asked for additional permissions on extension update. Does the dns permission trigger such a request for additional permissions?

Reply:
The permissions Firefox is asking for are the same as before making the change.